src/pytorch_lightning/utilities/types.py

```diff
@@ -131,6 +133,9 @@ def no_sync(self) -> Generator:
 LRSchedulerTypeTuple = (torch.optim.lr_scheduler._LRScheduler, torch.optim.lr_scheduler.ReduceLROnPlateau)
 LRSchedulerTypeUnion = Union[torch.optim.lr_scheduler._LRScheduler, torch.optim.lr_scheduler.ReduceLROnPlateau]
 LRSchedulerType = Union[Type[torch.optim.lr_scheduler._LRScheduler], Type[torch.optim.lr_scheduler.ReduceLROnPlateau]]
+LRSchedulerPLType = Union[
```

Comment:
added to support return type when `lr_schedulers` encounters:

```python
        if len(lr_schedulers) == 1:
            return lr_schedulers[0]
```

Reply:
I don't understand this

Reply:
cleaned in https://github.com/Lightning-AI/lightning/pull/13603/commits/205e546805833061d225c8237d3a39d40cbbd0dc